simpleSequential/test_isPositiveAndDivisibleBy69.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from isPositiveAndDivisibleBy69 import isPositiveAndDivisibleBy69
+
+class TestsIsPositiveAndDivisibleBy69(unittest.TestCase):
```

Comment:
Great job for listing all the combinations.

Reply:
Can you swap the test on line 11 12 and 14 15 so that we have a better order?

Positive + Divisble
Positive + NotDivisible
Negative + Divisible
Negative + NotDivisible

Some kind of like the follow convention:

True True
True False
False True
False False